src/databricksbundle/_config/config_test_daipe_help.yaml

```diff
@@ -0,0 +1,17 @@
+parameters:
+  daipehelpargs:
+    daipecore: daipe-core
+  databricksbundle:
+    databricks_connect:
+      connection:
+        address: 'https://dbc-123.cloud.databricks.com'
+        token: 'abcd123456'
+        cluster_id: '0416-084917-doles835'
+
+
```

Comment:
this whole file probably isn't necessary, we don't need a special config for testing, we can use config.yaml